pyaedt/modeler/Modeler.py

```diff
@@ -833,58 +833,27 @@ def set_working_coordinate_system(self, name):
         return True
 
     @aedt_exception_handler
-    def add_workbench_link(self, objects, ambient_temp=22, create_project_var=False, enable_deformation=True):
-        """Assign Temperature and Deformation Objects for WorkBench Link. From 2020R2 Material needs to have Thermal Modifier
+    def set_objects_deformation(self, objects):
```

Comment:
Add deprecation error to the old `add_workbench_link`.  Looks like we're completely removing it.

Reply:
Done.